applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_solver.py

```diff
@@ -52,8 +53,16 @@ def __init__(self, main_model_part, custom_settings):
             "model_import_settings": {
                 "input_type": "mdpa",
                 "input_filename": "unknown_name",
-                "perform_partitioning": true,
-                "input_file_label": 0
+                "perform_partitioning": true
+            },
+            "restart_settings" : {
+                "load_restart"                  : false,
+                "restart_load_file_label"       : "",
+                "save_restart"                  : false,
+                "restart_save_frequency"        : 1.0,
+                "restart_control_type"          : "time",
+                "move_restart_files_to_folder"  : true,
```

Comment:
Thanks, the previous implementation created a lot of files,  making impossible to work on the folder